Parent/Parent/Conversations/ConversationDetailCell.swift

```diff
@@ -16,142 +16,151 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
+import AVKit
 import UIKit
 import Core
 
 class ConversationDetailCell: UITableViewCell {
-    @IBOutlet weak var messageLabel: DynamicLabel!
-    @IBOutlet weak var fromLabel: DynamicLabel!
-    @IBOutlet weak var toLabel: DynamicLabel!
-    @IBOutlet weak var dateLabel: DynamicLabel!
+    @IBOutlet weak var messageLabel: UILabel!
+    @IBOutlet weak var fromLabel: UILabel!
+    @IBOutlet weak var toLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
```

Comment:
Why don't we want these to be DynamicLabel?

Reply:
`DynamicLabel` only provides stingly-typed helpers for Interface Builder. The swift code should _not_ use those helpers, since there are strongly typed better ones.